src/main/resources/swagger/ga4gh-workflow-execution.yaml

```diff
@@ -0,0 +1,289 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: CWL workflow execution service API
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+paths:
+  /jobs:
+    post:
+      summary: submit a new job
```

Comment:
Gotcha. The reason for that definition doc was that everybody in the batch-y type of space uses the same large handful of words to mean different things (e.g. in my world `job` is what doc describes as a `TaskInstance`). 

Prior to proceeding with merging we should discuss what the appropriate term is for this concept. I don't like using `job` (it was purposefully avoided previously) but following the previous pattern we'd end up with `WorkflowInstance` which seems unwieldy at best.